examples/script_nifti.py

```diff
@@ -1,10 +1,37 @@
+import os
+import pathlib
+import shutil
+import sys
 from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
```

Comment:
I always sort the stdlib before and separate from local dependencies:

```suggestion
import sys
import os
import pathlib
import shutil
import tempfile
import urllib.request as ur
from zipfile import ZipFile

from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
```